explorer/ast/declaration.cpp

```diff
@@ -342,14 +342,14 @@ auto FunctionDeclaration::Create(Nonnull<Arena*> arena,
                                  Nonnull<TuplePattern*> param_pattern,
                                  ReturnTerm return_term,
                                  std::optional<Nonnull<Block*>> body,
-                                 bool is_virtual)
+                                 VirtualOverride override)
```

Comment:
```suggestion
                                 VirtualOverride virt_override)
```

Reply:
Same, missing file save, thanks for catching it :)